app/views/products/show.html.haml

```diff
@@ -0,0 +1,28 @@
+%h1
+  =@product.name
+%div
+  %span.fw-lighter
+    Vendor code:
+  =@product.code
+.row.mt-2
+  .col-6
+    -if @product.image.attached? && @product.image.persisted?
```

Comment:
why do we need persisted check here?

Reply:
So somehow if we trying to add not image file(docx for example) it is anyway makes file attached and 
`if @product.image.attached?` returns true, but `persisted?` fixed this